lego/apps/events/models.py

```diff
@@ -1,5 +1,6 @@
+from __future__ import annotations
```

Comment:
I thought this would finally be mandatory in Python 3.11, but that was not the case ..